### PR TITLE
IG-8562: Fix labels hashing in Nuclio functions

### PR DESCRIPTION
**Pull request by imakhlin**
See http://jira.iguazeng.com:8080/browse/IG-8562 for details

### CODE REVIEW

**Comment by gtopper** [comment]
Triggering Travis...

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*.dylib
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+.glide/
```
Comment (gtopper) [inline .gitignore]: I thought we weren't going to use any package manager...

File: pkg/utils/labels.go [hunk 2]
```diff
@@ -32,6 +32,8 @@ const (
 	InstanceName = "instance"
 )
 
+var hash = xxhash.New()
```
Comment (gtopper) [inline pkg/utils/labels.go]: Shared state... Race❗️🐎

File: .gitignore (hunk 1 above)
Comment (gtopper) [reply .gitignore]: Better do in a separate PR if you want to add a `.gitignore` file. I actually kind of like the freedom to have my own `.gitignore` 🙊

**Comment by imakhlin** [comment]
```CMD: time V3IO_TSDBCFG_PATH="/home/iguazio/go/bin/v3io-test.yaml" TSDB_BENCH_INGEST_CONFIG="/home/iguazio/go/bin/tsdb-bench-test-config.yaml" go test -run=XXX -bench=^BenchmarkIngest$ -benchtime 1m -timeout 10m ../../test/benchmark
Setup: Dev8

---------------------------------------------- Original XXHASH implementation (WARM) --------------------------------------

/go/bin/tsdb-bench-test-config.yaml" go test -run=XXX -bench=^BenchmarkIngest$ -benchtime 1m -timeout 10m ../../test/benchmark
goos: linux
goarch: amd64
pkg: github.com/v3io/v3io-tsdb/test/benchmark
BenchmarkIngest-20       5000000             21625 ns/op
--- BENCH: BenchmarkIngest-20
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1
        BenchmarkIngest_test.go:77:
                Test complete. Count: 100
        BenchmarkIngest_test.go:77:
                Test complete. Count: 10000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1000000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 5000000
        ... [output truncated]
PASS
ok      github.com/v3io/v3io-tsdb/test/benchmark        129.895s

real    2m10.564s
user    3m26.770s
sys     0m20.804s

-------------------------------------------- New XXHASH (WARM) ---------------

goos: linux
goarch: amd64
pkg: github.com/v3io/v3io-tsdb/test/benchmark
BenchmarkIngest-20       5000000             22291 ns/op
--- BENCH: BenchmarkIngest-20
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1
        BenchmarkIngest_test.go:77:
                Test complete. Count: 100
        BenchmarkIngest_test.go:77:
                Test complete. Count: 10000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1000000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 5000000
        ... [output truncated]
PASS
ok      github.com/v3io/v3io-tsdb/test/benchmark        133.226s

real    2m13.888s
user    4m30.841s
sys     0m43.101s

--------------------------------------- New XXHASH (Cold) -----------------------------------------

goos: linux
goarch: amd64
pkg: github.com/v3io/v3io-tsdb/test/benchmark
BenchmarkIngest-20       3000000             23698 ns/op
--- BENCH: BenchmarkIngest-20
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1
        BenchmarkIngest_test.go:77:
                Test complete. Count: 100
        BenchmarkIngest_test.go:77:
                Test complete. Count: 10000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1000000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 3000000
        ... [output truncated]
PASS
ok      github.com/v3io/v3io-tsdb/test/benchmark        96.365s

real    1m36.957s
user    5m10.894s
sys     1m12.187s

----------------------------------- New XXHASH (WARM) ---------------------------------------------------
goos: linux
goarch: amd64
pkg: github.com/v3io/v3io-tsdb/test/benchmark
BenchmarkIngest-20       5000000             22958 ns/op
--- BENCH: BenchmarkIngest-20
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1
        BenchmarkIngest_test.go:77:
                Test complete. Count: 100
        BenchmarkIngest_test.go:77:
                Test complete. Count: 10000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 1000000
        BenchmarkIngest_test.go:77:
                Test complete. Count: 5000000
        ... [output truncated]
PASS
ok      github.com/v3io/v3io-tsdb/test/benchmark        136.593s

real    2m17.144s
user    5m19.243s
sys     0m59.427s```

**Pull request merged**